src/test/java/hlf/java/rest/client/integration/ConfigHandlerControllerIntegrationTest.java

```diff
@@ -44,17 +42,16 @@ static void setup() throws IOException {
 
   @Test
   void uploadConfigFile() throws Exception {
```

Comment:
Q: Is this test case relevant anymore?

Reply:
Oh.. This test has to be renamed. Thanks for pointing out.

Reply:
This is addressed.